app/lib/utils/queryMapper.js

```diff
@@ -22,42 +22,79 @@ function getLocationHeading(query) {
       return 'redirect';
     }
   }
-  return undefined;
+  return false;
 }
 
-function getResultsHeading(query, loc) {
-  if (loc && query.type) {
-    const location = loc.toUpperCase();
-    if (isProfessionalChoice(query)) {
-      return `Sexual health professionals near '${location}'`;
-    }
-    if (utils.areEqual(query.type, constants.serviceTypes.kit)) {
-      if (query.origin === constants.serviceChoices['16to25']) {
-        return `Where you can pick up a free test kit near '${location}'`;
-      } else if (query.origin === constants.serviceChoices.over25) {
-        return `Where you can buy a test near '${location}'`;
-      }
-    }
+function getResultsInfoForProfessionalsByAge(ageChoice, location) {
+  const correctResultsParams = true;
```

Comment:
These 3 `const`s aren't needed and could be set directly in the returned object, as some of the functions in this module do.